"0","data <- Latinobarometro_2023_Esp_v1_0
"
"2","Error: objeto 'Latinobarometro_2023_Esp_v1_0' no encontrado
"
